packages/components/psammead-brand/src/index.jsx

```diff
@@ -69,34 +65,26 @@ const StyledSpan = styled.span`
 const BrandSvg = styled.svg`
   display: block;
   height: ${SVG_HEIGHT};
-  width: ${SVG_WIDTH};
   margin-top: ${SVG_TOP_OFFSET};
-`;
-
-const SvgGroup = styled.g`
   fill: #fff;
-
   @media screen and (-ms-high-contrast: active), print {
     fill: windowText;
   }
 `;
 
-const Brand = ({ brandName }) => (
+const Brand = ({ brandName, svg }) => (
   <StyledWrapper>
     <ConstraintWrapper>
       <StyledLink href="https://www.bbc.co.uk/news">
         <StyledSpan>
           <BrandSvg
-            viewBox={VIEWBOX_VALUES}
+            viewBox={`0 0 ${svg.viewbox.width} ${svg.viewbox.height}`}
             xmlns="http://www.w3.org/2000/svg"
             focusable="false"
             aria-hidden="true"
+            width={(SVG_HEIGHT_PX * svg.ratio).toFixed(2)}
```

Comment:
Just noticed this means we will be doing height in `rem`s but width in `px`, can we do this using `rem`s too?